react-commenting.js

```diff
@@ -73,13 +78,15 @@ class Board extends Component{
   render(){
     const { gameBoard, currentPlayer, winner } = this.state
     // 7) Here:
+    //This return the map of an array with the squares to be rendered.
     let mappedGameBoard = gameBoard.map((value, index) => {
```

Comment:
Yup, it is mapping over the array from the state object and giving each index a <Square /> value - then saving that process in a variable.